spec/rubocop/cop/rails/strong_parameters_expect_spec.rb

```diff
@@ -92,9 +106,7 @@
 
       expect_correction(<<~RUBY)
         params.expect(
-          user: [:name, # comment
-          :age] # comment
-        )
+          user: [:name, :age])
```

Comment:
Could not find a solution to preserve comments when autocorrecting in a recursive fashionw